base_agent/dialogue_objects/filter_helper.py

```diff
@@ -174,6 +175,23 @@ def interpret_task_filter(interpreter, speaker, filters_d, get_all=False):
     return F
 
 
+def interpret_dance_filter(interpreter, speaker, filters_d, get_all=False):
```

Comment:
currently some filters are functions and some classes. long term, we should be consistent and pick one?